app/controllers/social_networking/profile_pages_controller.rb

```diff
@@ -59,12 +58,13 @@ def set_current_profile
       end
     end
 
-    def set_icon_size
-      @profile_page_flower_list_dimensions = Rails
-                                             .application
-                                             .config
-                                             .PROFILE_PAGE_LIST_DIMENSIONS
+    def profile_page_flower_list_dimensions
```

Comment:
I'm concerned about having this method in this engine, since it's specific to apps with flower icons. Can you think about how it could be refactored either to be more general, or to be moved into the host apps?